mava/configs/env/multicvrp.yaml

```
@@ -0,0 +1,14 @@
+# ---Environment Configs---
+defaults:
+  - _self_
+  - scenario: multicvrp-2v-20c # [multicvrp-2v-20c, multicvrp-2v-6c]
+
+env_name: MultiCVRP
+
+# Whether the environment observations encode implicit agent IDs. If True, the AgentID wrapper is not used.
+# This should not be changed.
+implicit_agent_id: False
+
+eval_metric: episode_return
+
+kwargs: {}
```

We can just set it to None if it is not getting used. 
```suggestion
kwargs: ~
```

---

it's not getting used but for consistency with other Jumanji env it needs to be there.
If it's set to ~ it will lead to an error on `train_env = jumanji.make(env_name, generator=generator, **config.env.kwargs)` due to attempting to unpack a Nonetype.
should i make the unpacking of kwargs optional? something like
```
if kwargs in config.env:
     train_env = jumanji.make(env_name, generator=generator, **config.env.kwargs)
else:
     train_env = jumanji.make(env_name, generator=generator)
```

---

Ah good point, let's just leave it as is in that case.